roles/openshift_node/tasks/openvswitch_system_container.yml

```diff
@@ -1,14 +1,11 @@
 ---
 - set_fact:
-    l_use_crio: "{{ openshift_docker_use_crio | default(false) }}"
-
-- set_fact:
     l_service_name: "cri-o"
-  when: l_use_crio
+  when: openshift_docker_use_crio
```

Comment:
This could probably be handled in role defaults as well, but we'll keep it for now.